openslides/assignments/static/js/assignments/site.js

```diff
@@ -89,6 +92,9 @@ angular.module('OpenSlidesApp.assignments.site', ['OpenSlidesApp.assignments'])
                                         return Assignment.loadRelations(assignment, 'agenda_item');
```

Comment:
I think our old code (down in the controller) is not correct. Assignment.loadRelations returns a promise which cannot be resolved until the controller is totally established in the DOM because the browser has only one thread/process for all JS stuff. The magic of `resolve` is that all promises are resolved before the site changes to the new state. So we can ensure that everything needed is already there. See #2295.